### PR TITLE
add basic CI & update conda env dependencies

### PR DESCRIPTION
**Pull request by lkeegan**
- set up `measelab` conda env
- install `mease-lab-to-nwb`
- add simple test that just checks we can import it

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Create measelab conda environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          auto-activate-base: false
+          python-version: 3.8
+          activate-environment: measelab
+          environment-file: mease-env.yml
```
Comment (alejoe91) [inline .github/workflows/ci.yml:25]: for the time being, I would point to the master versions of spikeextractors and neo until they are released ;)

Comment (lkeegan) [reply .github/workflows/ci.yml:25]: thanks @alejoe91 - I did this (also had to do the same for `nwb-conversion-tools` and add a couple of dependencies) and now it seems to be working

**Comment by alejoe91** [comment]
@lkeegan is this ready to merge? 

As a sidenote, we think that the best pipeline would be that you make your own fork of the repo, so you can continue working on it 'independently', i.e. without needing our approval to make changes. What do you think?

**Comment by lkeegan** [comment]
@alejoe91 yes this should be ready to merge.

And sure, I can make changes to my fork without also submitting PRs here if that woudl work better for you - either way is fine by me.

**Comment by alejoe91** [comment]
> @alejoe91 yes this should be ready to merge.

Merging then!
 
> And sure, I can make changes to my fork without also submitting PRs here if that woudl work better for you - either way is fine by me.

No need to make incremental PRs at this stage in my opinion. :)

**Pull request merged**